crates/ibc/src/applications/transfer/relay/on_recv_packet.rs

```diff
@@ -40,7 +40,10 @@ pub fn process_recv_packet_execute<Ctx: TokenTransferExecutionContext>(
             .get_channel_escrow_address(&packet.port_on_b, &packet.chan_on_b)
             .map_err(|token_err| (ModuleExtras::empty(), token_err))?;
 
-        ctx.send_coins(&escrow_address, &receiver_account, &coin)
+        ctx.send_coins_validate(&escrow_address, &receiver_account, &coin)
```

Comment:
this should go in a new `process_recv_packet_validate()` ([see comment](https://github.com/cosmos/ibc-rs/pull/512/files#r1131061591))

Reply:
actually it is correct as is, added clarifying comment [f7cde4f](https://github.com/cosmos/ibc-rs/pull/512/commits/f7cde4fd14de3ed0b6e449417797349fa5d8a5ca)